.gitlab-ci.yml

```diff
@@ -357,38 +353,14 @@ pup7.x-unit:
 # Repo-specific content
 # ==============================================================================
 
-pup5.pe:
-  <<: *pup_5_pe
-  <<: *acceptance_base
-  script:
-    - 'bundle exec rake beaker:suites[default]'
-
-pup5.pe-fips:
-  <<: *pup_5_pe
+pup7:
+  <<: *pup_7_x
   <<: *acceptance_base
   script:
-    - 'BEAKER_fips=yes bundle exec rake beaker:suites[default]'
+    - 'bundle exec rake beaker:suites[default,default]'
 
 pup6.x:
   <<: *pup_6_x
   <<: *acceptance_base
   script:
-    - 'bundle exec rake beaker:suites[default]'
-
-pup6.x-fips:
```

Comment:
Why were FIPS mode tests removed?

Reply:
Got a bit too aggressive in the pruning. Will restore one of them